libopenage/input/input_manager.h

```diff
@@ -150,8 +180,25 @@ class InputManager : public openage::InputHandler {
 	 */
 	coord::window mouse_position;
 	coord::window_delta mouse_motion;
-
 	friend InputContext;
+
+	std::map<std::string, int> stringToMod = {
```

Comment:
The `std::unordered_map` is hashed and way faster than the map which just is a search tree. Usage is the same, except you need `std::hash<key_type>` for the hashmap, the map only needs `operator <`